README.md

```diff
@@ -1,4 +1,35 @@
 # xfero
-File Transfer Orchestration software
+A practical File Transfer Orchestration solution for business
 
-This needs filling in
+## Why XferO?
+**THE MISSING LINK IN BUSINESS FILE TRANSFER SERVICES**
+
+XferO is a complete orchestration framework that simplifies complex file transfer integration for business file transfer services whilst giving the business community the ability to manage these services without having to engage with their technical department.
+
+## Whats different?
+The primary distinguishing features of xfero are:
+
+- Xfero puts the business community in charge. Providing a simple GUI for configuration of transfer routing.
```

Comment:
Replace with ... "XferO puts the business community in charge by providing a simple GUI for configuring and managing transfer routes."

Need to replace all with XferO if that is the chosen name.

Does this impact the code ?
Change the name of the "whatever xfero is in GitHub".